packages/falter-berlin-network-defaults/uci-defaults/freifunk-berlin-wireless-defaults

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+#
+## Setup a wifi for connecting to the router before
+## the first wizard run
+#
+
+. /lib/functions.sh
+. /lib/config/uci.sh
+. /lib/functions/system.sh
+
+mac_addr=$(get_mac_label | cut -c 10-)
+
+if [ -z "$mac_addr" ]; then
+    # get propable labels-MAC manually instead
+    mac_addr=$(ubus call network.device status | sed -n 's/.*macaddr": "\(.*\)".*/\1/p' |
+        sort | uniq | sed -n '2p' | cut -c 10-)
```

Comment:
We don't know that the first mac will always be 00:00:00:00:00:00.  So, maybe change this to
```
ubus call network.device status | sed -n 's/.*macaddr": "\(
.*\)".*/\1/p' | sort | uniq | grep -v "00:00:00:00:00:00" | sed -n '1p' | cut -c 10-
```

Reply:
right. Fixed that in the most recent commit.